extensions/amp-story/1.0/amp-story.js

```diff
@@ -2379,14 +2377,6 @@ export class AmpStory extends AMP.BaseElement {
       this.pages_[0].setState(PageState.NOT_ACTIVE);
       this.pages_[0].setState(PageState.PLAYING);
     }
-
-    // Reset all pages so that they are offscreen to right instead of left in
-    // desktop view.
-    switchPromise.then(() => {
-      this.pages_.forEach((page) =>
-        removeAttributeInMutate(page, Attributes.VISITED)
```

Comment:
`replay_()` is really trying to reset everything, including this, like a page reload. We should still remove the attributes

Reply:
In this PR I want to make the removal of animations more targeted:
- Current: first page of each story (independently of whether it was visited before or not).
- New: initial page of the story on the first visit of the page.

This means that when the story is replayed we don't need to skip the animations, especially because we're not on the initial page any more on some cases.

However, I understand your point so I'll re-implement this feature with better semantics (and probably remove this attribute altogether).